ember-app/app/mixins/issue-filters.js

```diff
@@ -0,0 +1,110 @@
+import Ember from 'ember';
+
+var IssueFiltersMixin = Ember.Mixin.create({
+  filters: Ember.inject.service(),
+
+  //Public Methods
+  isHidden: function(item){
+    var filters = this.filtersByMode(2);
+    return this.filter(filters, item);
+  },
+  isDim: function(item){
+    var filters = this.filtersByMode(1);
+    return this.filter(filters, item);
+  },
+
+  filter: function(filter_groups, item){
+    var inclusive_matches = this.runFilters(item, filter_groups, "inclusive");
+    var grouping_matches = this.runFilters(item, filter_groups, "grouping");
+
+    if(grouping_matches || inclusive_matches){ return true }
+    return false;
+  },
+
+  //Private Methods
+
+  //// Arrange Filters into Groups based on their strategy, sub-filtered by mode
+  // {
+  //   grouping: {
+  //     labels: []
+  //   },
+  //   inclusive: {
+  //     member: [],
+  //     board: []
+  //   }
+  // }
+  //
+  filtersByMode: function(mode){
```

Comment:
this is a terrible method, was only meant as a POC but I forgot to refactor. Should NOT be building complex objects n+1 times for each card render